src/main/java/api/service/SlackService.java

```diff
@@ -0,0 +1,23 @@
+package api.service;
+
+import org.aeonbits.owner.ConfigFactory;
+import org.json.simple.JSONObject;
+
+import static io.restassured.RestAssured.given;
+
+public class SlackService {
+    private static SlackConfig config = ConfigFactory.create(SlackConfig.class, System.getProperties());
+
+    public static void postNotification(String message) {
+        JSONObject request = new JSONObject();
+        request.put("text", message);
+        String postUrl = String.format("https://hooks.slack.com/services/%s", config.token());
+        given()
+                .header("Content-type", "application/json")
+                .body(request.toJSONString()).
+                when()
```

Comment:
move dot from 17th line to 18th

Reply:
I didn't pay the attention to these tests at all, cause this is not the goal of the HW. But nevertheless, fixed.